### ISSUE TITLE
Git config incorrect (http fetcher not working)

### ISSUE THREAD

**Issue by danielhams** [sgidevnet/sgug-rse#2]
Compared to regular didbs git cannot clone from https.

**Comment by danielhams** [comment]
Issue was with libcurl.so being unable to find libgcc_s.so - fixed in v0.0.2 - new SRPMs and RPMs available under releases.

Fix == use `-R%{_libdir}` and `-rpath %{_libdir}`

**Comment by danielhams** [comment]
Confirmed OK for me with 0.0.3:
```
[sgugshell dan@leafy tmpgitclone]$ git clone https://github.com/danielhams/didbs.git
Cloning into 'didbs'...
remote: Enumerating objects: 105, done.
remote: Counting objects: 100% (105/105), done.
remote: Compressing objects: 100% (77/77), done.
remote: Total 5716 (delta 50), reused 59 (delta 28), pack-reused 5611
Receiving objects: 100% (5716/5716), 6.98 MiB | 1.17 MiB/s, done.
Resolving deltas: 100% (4354/4354), done.
Checking out files: 100% (872/872), done.
```
